bigbluebutton-html5/imports/ui/components/video-dock/container.jsx

```diff
@@ -1,15 +1,26 @@
-import React from 'react';
+import React, { Component } from 'react';
 import { createContainer } from 'meteor/react-meteor-data';
 
 import VideoDock from './component';
+import VideoService from './service';
+import Users from '/imports/api/users';
 
-const VideoDockContainer = props => (
-  <VideoDock>
-    {props.children}
-  </VideoDock>
-);
+class VideoDockContainer extends Component {
+  constructor(props) {
+    super(props);
+  }
 
-export default createContainer(() => {
-  const data = {};
-  return data;
-}, VideoDockContainer);
+  render() {
+    return (
+      <VideoDock {...this.props}>
+        {this.props.children}
+      </VideoDock>
+    );
+  }
+}
+
+export default createContainer(() => ({
+  sendUserShareWebcam: VideoService.sendUserShareWebcam,
+  sendUserUnshareWebcam: VideoService.sendUserUnshareWebcam,
+  users: Users.find().fetch(),
```

Comment:
Could you move this and the Users import from above to the `video-dock/service.js`, please?

We have a convention where all the communication with the collections and Meteor server-side is done in the service.js files. So that it was easier to move data fetching somewhere else, if we need it.

Just add a function to the service.js:

```
import Users from '/imports/api/users';

...

const getAllUsers() {
  return Users.find().fetch();
}

...

export default {
  sendUserShareWebcam, sendUserUnshareWebcam, joinVideo, exitVideo, getAllUsers
};

```


And replace this line with `users: VideoService.getAllUsers(),`

P.S. just food for thought - shouldn't you specify something in the find query? Will it work with breakout rooms? (I'm not sure about the intended behavior, just asking) 😄